### ISSUE TITLE
Handling of missing type packages

### ISSUE THREAD

**Issue by bonchien** [aluanhaddad/aurelia-types-installer#1]
When typings don't exist the tool throws a large hard to decipher error block.  It would be nice to throw the error as a warning since the tool doesn't exit with an error.  Additionally it would be nice to be able to suppress the warnings on typings files that I don't expect to exist and to specify typings to ignore.  Great tool, thank you!

**Comment by aluanhaddad** [comment]
@bonchien, thank you for opening this issue!

You make excellent points and it is important to address all of them.

> It would be nice to throw the error as a warning since the tool doesn't exit with an error

Actually, these are warnings in the sense that the command still exits with a code of `0` so as to not fail builds. It is really verbose logging issue, but given that it very understandably causes confusion, and given that they are serialized stack traces, it should at least be under a `--verbose` flag with the default logging displaying a shorter, more helpful message.

> Additionally it would be nice to be able to suppress the warnings on typings files that I don't expect to exist and to specify typings to ignore.

A `--ignore` flag, taking a list of package ids could be added. Additionally, I think it would be good to display a list summarizing the results of tool, successes and failures, as well as an explanation of the implications of a failure.

I was thinking something like
```bash
$ ati install
installing aurelia typings...
summary:
installed 3 aurelia typings
- aurelia-bootstrapper
- aurelia-framework
- aurelia-templating-router

unable to locate typings for 3 aurelia packages:

- aurelia-bootstrap
- aurelia-dialog

try adding manually adding their src directories to "tsconfig.paths.json" or manually creating
typings using `declare module "aurelia-bootstrap" {...}`
```
Let me know what you think.

**Comment by aluanhaddad** [comment]
Fixed in [0.2.14](https://github.com/aluanhaddad/aurelia-types-installer/releases/tag/0.2.14)

I went a bit further than what was discussed here, providing multiple fallback suggestions and colorizing output.

Here is an example of the new output 

![image](https://user-images.githubusercontent.com/5034656/29487679-87c21a76-84cb-11e7-9176-94e64ac318ac.png)

Thanks again for raising this issue.
